realsense_camera/src/base_nodelet.cpp

```diff
@@ -218,8 +188,9 @@ namespace realsense_camera
     {
       // camera not found
       string error_msg = " - Couldn't find camera to connect with ";
-      error_msg += "serial_no = " + serial_no + ", ";
-      error_msg += "usb_port_id = " + usb_port_id;
+      error_msg += "serial_no = " + serial_no_ + ", ";
```

Comment:
This looks like it will have a confusing error message if there is one or more cameras connected of the wrong type, and none of the correct type.